<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book</class>
 <widget class="QWidget" name="Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="BookTitle"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>类别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="BookCategory"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="BookPress"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>出版年份</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="BookLowerYear">
      <property name="maximum">
       <number>3000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>————</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="BookUpperYear">
      <property name="maximum">
       <number>3000</number>
      </property>
      <property name="value">
       <number>3000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>282</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>价格</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="BookLowerPrice">
      <property name="maximum">
       <double>500.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>————</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="BookUpperPrice">
      <property name="maximum">
       <double>500.000000000000000</double>
      </property>
      <property name="value">
       <double>99.989999999999995</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="BookAuthor"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BookSearchPushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AllBookPushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询所有书目</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="BookTable">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>621</width>
     <height>491</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>书号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>类别</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>书名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版社</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>出版年份</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>作者</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>总藏书量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>目前库存量</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="BorrowPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>借阅</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>借书人</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="BorrowName"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="RefreshPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
